tests/integration/api_user_watch_test.go

```diff
@@ -23,8 +23,24 @@ func TestAPIWatch(t *testing.T) {
 
 	session := loginUser(t, user)
 	token := getTokenForLoggedInUser(t, session, auth_model.AccessTokenScopeReadUser)
+	tokenWithReadRepoScope := getTokenForLoggedInUser(t, session, auth_model.AccessTokenScopeReadRepository)
 	tokenWithRepoScope := getTokenForLoggedInUser(t, session, auth_model.AccessTokenScopeWriteRepository, auth_model.AccessTokenScopeReadUser)
 
+	t.Run("RepoSubscriptionUnauth", func(t *testing.T) {
+		defer tests.PrintCurrentTest(t)()
+
+		req := NewRequest(t, "GET", fmt.Sprintf("/api/v1/repos/%s/subscription", repo))
+		MakeRequest(t, req, http.StatusUnauthorized)
```

Comment:
Please move this into the `t.Run("IsWatching", func(t *testing.T) {` test. The other request is already tested there (but you could add yours there too because you use the read token).

Reply:
Should I move both tests to `IsWatching`?

Reply:
Yes, that the test for that endpoint.

Reply:
Sorry, I meant to move the `GET` calls into the `IsWatching` test in line 74.

Reply:
Ah, my mistake.